app/boards/arm/adv360pro/adv360pro.dtsi

```diff
@@ -0,0 +1,163 @@
+/*
+*
+* Copyright (c) 2021 Polarity Works
+* SPDX-License-Identifier: MIT
+*
+*/
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+#include <dt-bindings/led/led.h>
+#include <dt-bindings/zmk/matrix_transform.h>
+
+#include "adv360pro_pinctrl.dtsi"
+
+/ {
+    model = "Adv360";
+    compatible = "kinesis,adv360pro";
+
+    chosen {
```

Comment:
Usually our boards assign a default `zephyr,console` to a USB CDC ACM mode for easy USB logging. Intentionally omitted?

Reply:
I think this was dropped when i disabled logging on the fork, will add it back if it's convention

Reply:
Please. It's expected at this point for boards in ZMK.